src/common/backend/utils/cache/catcache.cpp

```diff
@@ -3296,3 +3296,29 @@ SetCatCacheList(CatCache *cache,
 
     DLAddHead(&cache->cc_lists, &cl->cache_elem);
 }
+
+/*
+ *	RelationHasCachedLists
+ *
+ *	Returns true if there is a catalog cache associated with this
+ * 	relation which is currently caching at least one list.
+ */
+bool RelationHasCachedLists(Relation relation)
```

Comment:
is Relation cheap enough to copy? or should we take it by const&

Reply:
Postgres uses it in this way and it should be pretty cheap. However, it is no harm to change to const&. Changed as suggested. Thanks.